docs/03_server/11_integration/07_data-pipeline/01_introduction.mdx

```diff
@@ -10,21 +10,90 @@ tags:
   - introduction
 ---
 
-Genesis Data Pipelines enable you to stream data into and out of your Genesis application. If you are looking to integrate your application with an external database or file system, this module should be your first consideration. If Data Pipeline does not offer you the solution you need, we suggest using [Apache Camel](../../../../server/integration/apache-camel/introduction/) to create the integration. 
+Genesis Data Pipelines enable you to stream data into and out of your Genesis application. If you are looking to integrate your application with an external database or file system, this module should be your first consideration. If Data Pipeline does not offer you the solution you need, we suggest using [Apache Camel](/server/integration/apache-camel/introduction/) to create the integration. 
 
 Each Data Pipeline defines a source, some mapping operations and a sink.
 
 | term | description |
 | ---- | ----------- |
 | source | Changes in data at this location trigger your Data Pipeline |
-| map | Transforms data from the data source |
+| operators | Transform data from the data source |
 | sink | Does something with the mapped data from the data source |
```

Comment:
I wasn't sure if this statement here needed to be more specific.:
>Does something with the mapped data from the data source

As I don't know much about it myself, best to double check here.